test/molecule-role/molecule/default/prepare.yml

```diff
@@ -133,12 +148,35 @@
   vars_files:
     - common_vars.yml
   tasks:
+    - name: Add testagent.com to /etc/hosts
+      become: true
+      lineinfile:
+        path: /etc/hosts
+        line: "{{ hostvars['receiver']['ansible_host'] }} test-stackstate-agent.sts"
     - name: Install stackstate-agent
       shell: "curl -o- {{agent_repo_url}}/install.sh | bash"
       environment:
         STS_API_KEY: "API_KEY"
         STS_URL: "http://{{ hostvars['receiver']['ansible_host'] }}:7077/stsAgent"
         CODE_NAME: "{{agent_current_branch}}"
+    - name: Replace STS URL to https URL
```

Comment:
The `STS_URL` is passed as environment variable to the installation script in the above task, can't we change it there directly ?

Reply:
yeah changing the URL here leads to `x509: certificate signed by unknown authority` error which will be in the logs until restarted with `SKIP_SSL_VALIDATION: true` flag.